test/test_jit_llga_fuser.py

```diff
@@ -481,6 +482,49 @@ def forward(self, x, y):
             graph, _ = self.checkTrace(m, [x, y])
 
 
+class TestEnableDisableLlgaFuser(JitTestCase):
+    def setUp(self):
+        super().setUp()
+        self.is_enabled = torch._C._jit_set_llga_enabled(False)
+
+    def tearDown(self):
+        torch._C._jit_set_llga_enabled(self.is_enabled)
+        super().tearDown()
+
+    def test_context_manager_test(self):
+        x = torch.randn(4, 8)
+        y = torch.randn(4, 8)
+        with torch.jit.fuser('fuser3'):
+            with torch.jit.fuser('fuser3'):
+                
+                def t1(x, y):
+                    o = x + y
+                    o = o + 2.0
+                    return o
+                t_jit = torch.jit.script(t1)
+                t_jit(x, y)
+                t_jit(x, y)
+                self.assertGraphContains(t_jit.graph_for(x, y), LLGA_FUSION_GUARD)
```

Comment:
May I know if it is possible to check
```suggestion
                self.assertGraphContains(t_jit.graph_for(x, y), LLGA_FUSION_GROUP)
```
If it works, since `LLGA_FUSION_GROUP` is already define in this file, you don't need to add a new string `LLGA_FUSION_GUARD`.

Reply:
The test passes with changing to FUSION_GROUP. 

What's the right/preferred usage for FUSION_GROUP and FUSION_GUARD?
I referred this test for fuser2 which uses FUSION_GUARD: https://github.com/wenzhe-nrv/pytorch/blob/737dbfe3b28e799603b1b992d170df4cbfb28c20/test/test_jit_cuda_fuser.py#L4902

Reply:
Either of them should be fine.
Since `LLGA_FUSION_GROUP` is already defined, we might not need to introduce another string.

Taken the below graph as an example, when the oneDNN graph fuser functions as expected, both `prim::oneDNNFusionGuard` and `prim::oneDNNFusionGroup` can be found in the graph.
```
graph(%self : __torch__.torch.nn.modules.pooling.___torch_mangle_379.AvgPool2d,
      %input : Tensor):
  %2 : Tensor, %3 : bool = prim::oneDNNFusionGuard[types=[Float(1, 4, 18, 18, strides=[1296, 324, 18, 1], requires_grad=0, device=cpu)]](%input)
  %4 : Tensor = prim::If(%3)
    block0():
      %5 : Tensor = prim::oneDNNFusionGroup_0[output_layouts=[0]](%2)
      -> (%5)
    block1():
      %6 : Function = prim::Constant[name="fallback_function", fallback=1]()
      %7 : (Tensor) = prim::CallFunction(%6, %input)
      %8 : Tensor = prim::TupleUnpack(%7)
      -> (%8)
  return (%4)
with prim::oneDNNFusionGroup_0 = graph(%input : Float(1, 4, 18, 18, strides=[1296, 324, 18, 1], requires_grad=0, device=cpu)):
  %1 : int[] = prim::Constant[value=[5, 5]]()
  %2 : int[] = prim::Constant[value=[1, 1]]()
  %3 : int[] = prim::Constant[value=[0, 0]]()
  %4 : bool = prim::Constant[value=0]()
  %5 : bool = prim::Constant[value=1]()
  %6 : NoneType = prim::Constant()
  %7 : Float(1, 4, 14, 14, strides=[784, 196, 14, 1], requires_grad=0, device=cpu) = aten::avg_pool2d(%input, %1, %2, %3, %4, %5, %6) # /home/chunyuan/TE/pytorch/torch/nn/modules/pooling.py:623:0
  return (%7)
```